DisCatSharp.Docs/articles/basics/web_app.md

```diff
@@ -8,18 +8,41 @@ Install the following packages:
  - DisCatSharp
  - DisCatSharp.Hosting
 
+## Notes
+Please be aware that this approach relies on Dependency Injection. You can either use one of Microsoft's default project templates for .Net Core Web App, or get a head start by using the
+`DisCatSharp.Hosting.ProjectTemplates` pack which contains a Bot Template to jumpstart your development. If you do the latter, majority of this is done for you.
```

Comment:
```suggestion
> [!Note]
Please be aware that this approach relies on Dependency Injection. You can either use one of Microsoft's default project templates for .Net Core Web App, or get a head start by using the
`DisCatSharp.Hosting.ProjectTemplates` pack which contains a Bot Template to jumpstart your development. If you do the latter, majority of this is done for you.
```

Reply:
We could use warn there, but I'm not sure

Reply:
I tried getting tabbed documentation working but had no luck. Even copied their stuff directly from their docs -- didn't work.

big sad

Reply:
Added some notes / important sections in there

Reply:
Yeah I've been fighting with this thing too yesterday